docker/Dockerfile

```diff
@@ -118,5 +118,5 @@ RUN \
     build-locale-archive
 
 WORKDIR /
-ENTRYPOINT ["/usr/bin/contextBroker","-fg"]
-EXPOSE 1026
\ No newline at end of file
+ENTRYPOINT ["/usr/bin/contextBroker","-fg", "-multiservice"]
```

Comment:
Question: wasn't multiservice enabled by default on later versions of Orion? Or maybe I just heard about it